src/app/components/customers/customers.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup } from '@angular/forms';
+import { SidebarService } from '../sidebar/sidebar.service';
+import { Headers } from 'src/app/models/tabledata.model';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
```

Comment:
sort imports, first system imports, then app imports
```suggestion
import { Component, OnInit } from '@angular/core';
import { FormBuilder, FormControl, FormGroup } from '@angular/forms';
import { HttpClient } from '@angular/common/http';
import { Observable } from 'rxjs';

import { SidebarService } from '../sidebar/sidebar.service';
import { Headers } from 'src/app/models/tabledata.model';
```